src/ParametricUtils.jl

```diff
@@ -775,47 +775,5 @@ function createMvNormal(v::DFGVariable, key=:parametric)
 end
 
 
-function autoinitParametric!(dfg::AbstractDFG,
```

Comment:
This is still under development and was changed in #1594. I'm also working on it locally as it's not reliable at all. Moving files around makes it a manual process to track changes and manage conflicts which I do not have time for now.